samples/templates/clouds-secrets/configs/clouds.yaml

```diff
@@ -0,0 +1,10 @@
+clouds:
+  os1pi020:
+    auth:
+      username: E415_s_os1pi020
+      password: KJFs!3CuCu%kjYuyfd87
```

Comment:
hope this is not your real password @sbueringer   :)

Reply:
Thought I removed it but I will change it 😀